src/js/containers/aboutTheData/modals/PublicationDatesContainer.jsx

```diff
@@ -57,24 +64,21 @@ const PublicationDatesContainer = ({
         }
     };
 
+    const createMillisecondDates = (data) => data.map((datesObj) => {
```

Comment:
can we move this out of the component definition?

Reply:
possible rename:

`convertDatesToMilliseconds`

Reply:
we can, just did not see any reason to